node/bft/src/lib.rs

```diff
@@ -49,9 +49,9 @@ pub const CONTEXT: &str = "[MemoryPool]";
 pub const MEMORY_POOL_PORT: u16 = 5000; // port
 
 /// The maximum number of milliseconds to wait before proposing a batch.
-pub const MAX_BATCH_DELAY_IN_MS: u64 = 2500; // ms
+pub const MAX_BATCH_DELAY_IN_MS: u64 = 5500; // ms
 /// The minimum number of seconds to wait before proposing a batch.
-pub const MIN_BATCH_DELAY_IN_SECS: u64 = 1; // seconds
+pub const MIN_BATCH_DELAY_IN_SECS: u64 = 5; // seconds
```

Comment:
A fixed delay of 5 seconds per proposal will always result in a block time that is >10 seconds, which is incorrect. Increasing this wait-time arbitrarily will **BREAK validator nodes** that are syncing near tip, because they will see existing blocks that are accurate with 1 sec wait times and believe they are invalid

The current wait time of 1 second already does not incur a blocktime of 2 seconds, because of all the overhead required to craft the block itself. 

The better approach is to keep the fixed minimum at something low (like 1 second) and introduce an additional wait-time that honest validators will follow based on the current network load. (This could in theory be a naive 5 seconds).